### PR TITLE
Replace ` ` with $( ) inside strings and in commands

### PR DESCRIPTION
**Pull request by abdelkafiahmed**
Replaced `  ` with $( ) because it's the new standard in shell scripting

### CODE REVIEW

File: lib/update/update.sh [hunk 1]
```diff
@@ -64,7 +64,7 @@ bpkg_update () {
         fi
         ;;
     esac
-  done
+  done }
```
Comment (jwerle) [inline lib/update/update.sh]: why wrap this in `{}` ?

File: lib/json/test/valid/string_in_object.json [hunk 2]
```diff
@@ -1 +1,3 @@
-{"key":"hello this is a string"}
\ No newline at end of file
+{
```
Comment (jwerle) [inline lib/json/test/valid/string_in_object.json]: why modify these test files?

File: lib/update/update.sh (hunk 1 above)
Comment (abdelkafiahmed) [inline lib/update/update.sh]: I forgot to remove the {}
You're right, it doesn't make any sense.

File: lib/json/test/valid/string_in_object.json (hunk 2 above)
Comment (abdelkafiahmed) [inline lib/json/test/valid/string_in_object.json]: It's the way JSON files needs to be formatted according to the convention.
+ it doesn't interfere with the content of the JSON file.

Comment (jwerle) [inline lib/json/test/valid/string_in_object.json]: I understand that. However, the JSON implementation is an upstream of https://github.com/dominictarr/JSON.sh and should not be modified.

Comment (abdelkafiahmed) [inline lib/json/test/valid/string_in_object.json]: Oh, I see. 
I didn't know that. Maybe you should mention it in credit section in the ReadMe file !
I'll drop the changes to the JSON files.

**Comment by jwerle** [comment]
@abdelkafiahmed thank you!

File: lib/init/init.sh [hunk 3]
```diff
@@ -120,7 +120,8 @@ required () {
     "SCRIPTS"
   do
     eval local val="\${${key}}"
-    [ -z "${val}" ] && error "Missing \`${key}' property"
+    [ -z "${val}" ] && error "Missing \`
+    ${key}' property"
```
Comment (dzuelke) [inline lib/init/init.sh]: is this line break intentional?

Comment (abdelkafiahmed) [inline lib/init/init.sh]: No it's not intentional !! My bad !

Comment (abdelkafiahmed) [inline lib/init/init.sh]: No it's not intentional !! My bad !

**Pull request merged**